LayoutTests/platform/mac/TestExpectations

```diff
@@ -2315,29 +2315,22 @@ imported/w3c/web-platform-tests/xhr/event-timeout-order.any.worker.html [ Failur
 
 webkit.org/b/237166 [ BigSur+ ] webgl/pending/conformance/textures/misc/tex-image-video-repeated.html [ Pass Timeout ]
 
-http/tests/webgpu/webgpu/idl/constants/flags.html [ Skip ]
-http/tests/webgpu/webgpu/idl/exposed.https.html [ Skip ]
-http/tests/webgpu/webgpu/idl/exposed.http.html [ Skip ]
```

Comment:
Can we mark these as [ Pass ] so when we do the next CTS import it's clear which tests we expect to pass vs which tests are new and we have no information about whether they pass or not?

Reply:
Yes good idea